source/partials/next_release/_release.md.erb

```diff
@@ -1,50 +1,46 @@
+  * <%= link_to_commit '69e4b3937b5d3a52cb4444f6caba17c4e1573ff3', 'Render the dashboard again after backup is completed', 5391 %>
+  * <%= link_to_commit '98853a28dd22f9094f513b4ef17190a2de1943ec', 'Fix a NullPointerException in the logs when a pipeline that is building is removed from the config file.' %>
+  * <%= link_to_commit 'aabbf06e969b62f0d73d1e091d1c88ae5ef9a258', 'Fix 500 server error when calling pipeline config api by ensuring that the default artifact directory is not null', 5342 %>
+  * <%= link_to_commit '6fddce6e290391e63992ec6195594f3149200362', 'Prevent console logs of cancelled jobs running on elastic agents from being lost', 5325 %>
+  * <%= link_to_commit 'e95e88b1dcc1a4337254874af929b8690a522163', 'Agent will ensure that plugins are loaded before performing any jobs.', 5169 %>
+  * <%= link_to_commit '49d741308cbbb499554e4e2843f0812e2e9b8460', 'Do not send email notification if user account is disabled.', 5279 %>
+  * <%= link_to_commit '13be966d353818be2c56ad00be16661e5257263d', 'Reenable pipeline operation buttons on dashboard if pipeline trigger fails.', 5153 %>
+
 
 <h4>Deprecations</h4>
 
-  * The alpine 3.5 based docker image for [gocd agent](https://hub.docker.com/r/gocd/gocd-agent-alpine-3.5/) has been deprecated and will not be built after November 2018 release of GoCD. This will be done as alpine 3.5 will reach [end-of-life](https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases) in November 2018.
    * With the introduction of AES for encryption/decryption in release 18.7.0, we are deprecating support for DES. Support for DES is retained to support DES cipher-texts configured in config-repos and will be available till the last release of GoCD for the year 2018. The <%= link_to_versioned_api '18.8.0', 'encryption', 'Encryption API' %> is updated to return a AES cipher-text, users are encouraged to update their DES cipher-texts in config-repos using this API.
-  * The Pipeline Config API v5 has been deprecated in release 18.7.0. This version of the API will be removed in a release scheduled for November 2018. <%= link_to_versioned_api '18.8.0', 'pipeline-config', 'Version 6' %> of the API is available, and users are encouraged to use it.
-  * The Template Config API v3 has been deprecated in release 18.7.0. This version of the API will be removed in a release scheduled for November 2018. <%= link_to_versioned_api '18.8.0', 'template-config', 'Version 4' %> of the API is available, and users are encouraged to use it.
+
 
 <h4>Breaking Changes</h4>
 
-  * We have stopped building ubuntu 12.04 based docker image for [gocd agent](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-12.04/) as ubuntu 12.04 is now end-of-life.
+  * The alpine 3.5 based docker image for [gocd agent](https://hub.docker.com/r/gocd/gocd-agent-alpine-3.5/) has been removed since alpine 3.5 has reached end of life.
```

Comment:
I remember having a different section for **docker** in the previous release notes. Should we move this to docker section?